src/dwi/tractography/resampling/arc.h

```diff
@@ -64,12 +66,12 @@ namespace MR {
             Arc (const size_t n, const point_type& s, const point_type& w, const point_type& e) :
                 nsamples (n),
                 start (s),
-                mid (0.5*(s+e)),
+                mid (w),
```

Comment:
> I borked the arc sampling all the way back in #555. Arc initialisation was completely ignoring the midpoint provided by the user. :-/

Ah, so this is essentially the fix for that, then...? But I still don't get why it looked like it was sampling along a completely different arc, if presumably all that happened was that the midpoint specified by the user was ignored, and the default midpoint used instead... (?)

Reply:
If I had to guess, I'd say that in `init_arc()`, it's solving for `c` as the centre point of a circle containing the three points provided by the user, but when the midpoint is the bisection point the circle can't be defined, so Eigen is setting `c` to `[0.0, 0.0, 0.0]`, and that's therefore the real-space point that all the sampling planes are passing through. But I didn't bother actually interrogating it that deeply, since it was clearly an egregious fault.

Reply:
OK, I guess that makes sense. Happy for it to be merged.